test/python/linear_attention/test_la.py

```
@@ -711,6 +755,319 @@ def test_bwd_with_checkpoints(backend, variant):
+def test_bwd_state_grad_dtype(backend, variant, state_dtype):
+    """The state gradients ride the state's own dtype end to end, asserted on the
+    raw op rather than through autograd."""
+    case = make_case(variant, torch.bfloat16, T=128, H=2)
+    state0 = (torch.randn(case.N, case.HO, case.V, case.K, device="cuda") * 0.05).to(state_dtype)
+    args = [to_thd(case.q), to_thd(case.k), to_thd(case.v), to_thd(case.gates["g"]), to_thd(case.gates["beta"])]
+    if variant == "gdn2":
+        args.append(to_thd(case.gates["w"]))
+    dO = torch.randn(case.T, case.HO, case.V, device="cuda", dtype=case.dtype)
+    dfs = (torch.randn(case.N, case.HO, case.V, case.K, device="cuda") * 0.1).to(state_dtype)
+    with waive_unsupported(backend, variant):
+        o, fs = pinned_op(backend, variant)(*args, case.cu, initial_state=state0, output_final_state=True)
```

Ruff (0.16.2)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)